spec.emu

```diff
@@ -76,36 +160,44 @@ contributors: Ujjwal Sharma, Younies Mahmoud
         1. Let _result_ be a new empty List.
-        1. If _durationFormat_.[[Style]] is `"digital"`, then
-          1. <mark>TODO: [[formats]].[[digital]] needs to be keyed by which fields are available in this case.</mark>
-          1. Let _template_ be _dataLocaleData_.[[formats]].[[digital]].[[&lt;_pr_&gt;]].
-          1. Let _patternParts_ be PartitionPattern(_pattern_).
-          1. For each element _patternPart_ in _patternParts_, in List order, do
-            1. Let _p_ be _patternPart_.[[Type]].
-            1. If _p_ is `"literal"`, then
-              1. Append _patternPart_ to the end of _result_.
-            1. Else if _duration_.[[&lt;_p_&gt;]] is not *undefined* and _p_ exists in _durationFormat_.[[Fields]], then
-              1. Let _num_ be FormatNumeric(_durationFormat_.[[NumberFormat]], _duration_.[[&lt;_p_&gt;]]).
-              1. Append the new Record { [[Type]]: _p_, [[Value]]: _num_ } to the end of _result_.
-        1. Else,
-          1. For each row in <emu-xref href="#table-duration-components"></emu-xref>, except the header row, in table order, do
-            1. Let _field_ be the Property value.
-            1. Let _val_ be _duration_.[[&lt;_field_&gt;]].
-            1. If _val_ is not *undefined* and _field_ exists in _durationFormat_.[[Fields]], then
-              1. Let _num_ be ! PartitionNumberPattern(_durationFormat_.[[NumberFormat]], _val_).
-              1. Let _pr_ be ! ResolvePlural(_durationFormat_.[[PluralRules]], _val_).
-              1. Let _dataLocale_ be _durationFormat_.[[DataLocale]].
-              1. Let _localeData_ be %DurationFormat%.[[LocaleData]].
-              1. Let _dataLocaleData_ be _localeData_.[[&lt;_dataLocale_&gt;]].
-              1. Let _template_ be _dataLocaleData_.[[formats]].[[&lt;_style_&gt;]].[[&lt;_field_&gt;]].[[&lt;_pr_&gt;]].
-              1. Let _parts_ be ! MakePartsList(_template_, _field_, _num_).
-              1. Let _concat_ be an empty String.
-              1. For each element _part_ in _parts_, in List order, do
-                1. Set _concat_ to the string-concatenation of _concat_ and _part_.[[Value]].
-                1. If _part_ has a [[Unit]] field, then
-                  1. Set _concat_ to the string-concatenation of _concat_ and _part_.[[Unit]].
-              1. Append the new Record { [[Type]]: _field_, [[Value]]: _concat_ } to the end of _result_.
-          1. Set _result_ to ! CreatePartsFromList(_durationFormat_.[[ListFormat]], _result_).
+        1. For each row in <emu-xref href="#table-duration-components"></emu-xref>, except the header row, in table order, do
+          1. Let _styleSlot_ be the Style Slot value.
+          1. Let _displaySlot_ be the Display Slot value.
+          1. Let _valueSlot_ be the Value Slot value.
+          1. Let _unit_ be the Unit value.
+          1. Let _style_ be _durationFormat_.[[&lt;_styleSlot_&gt;]].
+          1. If _style_ is `"numeric"`, then
+            1. Skip to the next iteration.
```

Comment:
Why does it skip to the next iteration for numeric styles? For numeric-styled units, where is the record appended to the parts list?

I admit I don't fully understand parts of this algorithm, so I'll ask a lot of dumb questions in my review. 😄